configs/cat_seg/catseg_vitb-r101_4xb1-warmcoslr2e-4-adamw-80k_ade20k-384x384.py

```diff
@@ -0,0 +1,100 @@
+_base_ = [
+    '../_base_/default_runtime.py', '../_base_/schedules/schedule_80k.py',
+    '../_base_/datasets/ade20k_384x384.py'
+]
+
+norm_cfg = dict(type='SyncBN', requires_grad=True)
+crop_size = (384, 384)
+data_preprocessor = dict(
+    type='SegDataPreProcessor',
+    size=crop_size,
+    # due to the clip model, we do normalization in backbone forward()
+    bgr_to_rgb=True,
+    pad_val=0,
+    seg_pad_val=255)
+# model_cfg
+model = dict(
+    type='EncoderDecoder',
+    data_preprocessor=data_preprocessor,
+    backbone=dict(
+        type='CLIPOVCATSeg',
+        feature_extractor=dict(
+            type='ResNet',
+            depth=101,
+            # only use the first three layers
+            num_stages=3,
+            out_indices=(0, 1, 2),
+            dilations=(1, 1, 1),
+            strides=(1, 2, 2),
+            norm_cfg=norm_cfg,
+            norm_eval=False,
+            style='pytorch',
+            contract_dilation=True,
+            init_cfg=dict(
+                type='Pretrained', checkpoint='torchvision://resnet101'),
+        ),
+        train_class_json='data/ade150.json',
+        test_class_json='data/ade150.json',
+        clip_pretrained='ViT-B/16',
+        clip_finetune='attention',
+    ),
+    neck=dict(
+        type='CATSegAggregator',
+        appearance_guidance_dim=1024,
+        num_layers=2,
+    ),
```

Comment:
I'm trying to align the inference performance with the official checkpoint., and I found it should set `pooling_size` to (1, 1).
https://github.com/KU-CVLAB/CAT-Seg/blob/2a780cce2db1d9881c0a26c6eb7790405be5462a/eval.sh#L37

Reply:
I'll try to find all the differences today, it may not be able to merge so quickly.

Reply:
> I'm trying to align the inference performance with the official checkpoint., and I found it should set `pooling_size` to (1, 1). https://github.com/KU-CVLAB/CAT-Seg/blob/2a780cce2db1d9881c0a26c6eb7790405be5462a/eval.sh#L37
I have fixed the configs.

Reply:
> I'll try to find all the differences today, it may not be able to merge so quickly.

I understand! Please take your time. Just @me for further problems. 👍